ash/src/vk/definitions.rs

```diff
@@ -33351,36 +29671,14 @@ unsafe impl<'a> TaggedStructure for PipelineExecutableInternalRepresentationKHR<
         StructureType::PIPELINE_EXECUTABLE_INTERNAL_REPRESENTATION_KHR;
 }
 impl<'a> PipelineExecutableInternalRepresentationKHR<'a> {
-    #[inline]
-    pub fn name(mut self, name: &CStr) -> core::result::Result<Self, CStrTooLargeForStaticArray> {
-        write_c_str_slice_with_nul(&mut self.name, name).map(|()| self)
-    }
     #[inline]
     pub fn name_as_c_str(&self) -> core::result::Result<&CStr, FromBytesUntilNulError> {
         wrap_c_str_slice_until_nul(&self.name)
     }
     #[inline]
-    pub fn description(
-        mut self,
-        description: &CStr,
-    ) -> core::result::Result<Self, CStrTooLargeForStaticArray> {
-        write_c_str_slice_with_nul(&mut self.description, description).map(|()| self)
-    }
-    #[inline]
     pub fn description_as_c_str(&self) -> core::result::Result<&CStr, FromBytesUntilNulError> {
         wrap_c_str_slice_until_nul(&self.description)
     }
-    #[inline]
-    pub fn is_text(mut self, is_text: bool) -> Self {
-        self.is_text = is_text.into();
-        self
-    }
-    #[inline]
-    pub fn data(mut self, data: &'a mut [u8]) -> Self {
-        self.data_size = data.len();
-        self.p_data = data.as_mut_ptr().cast();
-        self
-    }
```

Comment:
Edge-case: this is an out-array that the caller would have to allocate, by calling [`vkGetPipelineExecutableInternalRepresentationsKHR()`](https://registry.khronos.org/vulkan/specs/latest/man/html/vkGetPipelineExecutableInternalRepresentationsKHR.html) twice with this set to `NULL` initially, and set to a valid mutable slice on the second call.

Reply:
There appear to be 5 removed `&'a mut T`s in this PR: those all need to remain for this change o be convenient.

Reply:
Good catch. Can we just check for mutable ref arguments?

Also, you previously noted:

> note that upstream does not set returnedonly="true" for structs that require the user to allocate storage

Is that not true for these? Is the XML inconsistent here?